ouroboros-network/test/Test/Ouroboros/Network/PeerSelection/RootPeersDNS.hs

```diff
@@ -707,6 +728,18 @@ prop_local_updatesDomainsCorrectly mockRoots@(MockRoots lrp _ _ _)
               (tail tr)
      in property (fst r)
 
+prop_unit_4491 :: Property
+prop_unit_4491 =
+  prop_local_updatesDomainsCorrectly
+    (MockRoots [ (2, Map.fromList [(RelayAccessDomain "dns1" 00001, DoAdvertisePeer)
+               , (RelayAccessDomain "dns2" 00000, DoAdvertisePeer)])
+               ]
+               (singletonScript (Map.fromList [ ("dns1", [("0.0.0.0", 0)])
+                                              , ("dns2", [("0.0.0.1", 0)])]))
+               Map.empty
+               (singletonScript Map.empty))
+    (singletonScript (DNSTimeout 1000))
+    (singletonScript (DNSLookupDelay 0))
```

Comment:
Is this covered by `prop_local_updatesDomainsCorrectly`? If so, then do we need this?

Reply:
ugh I responded via email and it ended up in a different place. Actual reply:

It is now, I just used it to make sure we were covering the case that would
lead to the bug. I can remove it!

Reply:
Done!